### PR TITLE
Grafana Service Account Detector

### PR DESCRIPTION
**Pull request by ankushgoel27**
Created the grafana service account token detector

### CODE REVIEW

File: pkg/detectors/rubygems/rubygems.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://rubygems.org/api/v1/gems", nil)
```
Comment (0x1) [inline pkg/detectors/rubygems/rubygems.go:50]: i believe these changes on rubygems are duplicate, and were already merged in on the PR: https://github.com/trufflesecurity/trufflehog/pull/1781

Comment (ankushgoel27) [reply pkg/detectors/rubygems/rubygems.go:50]: Yes, thats correct. They are indeed duplicate

Comment (0x1) [reply pkg/detectors/rubygems/rubygems.go:50]: could you remove those changes from this PR pls

Comment (ankushgoel27) [reply pkg/detectors/rubygems/rubygems.go:50]: i tried removing these changes, can you please confirm.

If not, then i will close this PR and open another one

**Pull request closed without merging**